src/com/epam/homework/lists/StudentList.java

```diff
@@ -0,0 +1,84 @@
+package com.epam.homework.lists;
+
+public class StudentList {
+    private static final int DEFAULT_CAPACITY = 10;
+
+    private int size = 0;
+
+    private Student[] students;
+
+    public StudentList() {
+        students = new Student[DEFAULT_CAPACITY];
+    }
+
+    public StudentList(int capacity) {
+        if (capacity < 0) {
+            throw new IllegalArgumentException("Illegal capacity: " + capacity);
+
+        }
+        students = new Student[capacity];
+    }
+
+    public void add(Student student) {
+        ensureCapacity();
+        students[size] = student;
+        size++;
+    }
+
+    public void remove(Student student) {
+        for (int i = 0; i < students.length; i++) {
+            if (student.equals(students[i])) {
+                if (i == students.length - 1) {
+                    students[i] = null;
+                    size--;
+                } else {
+                    removeFromStudents(i);
+                }
+            }
+        }
+    }
+
+    public int size() {
+        return size;
+    }
+
+    public Student get(int index) {
```

Comment:
What is the purpose of separating cases: `index >= size` and `index < 0`?

Reply:
Just different error messages.

Reply:
Merged this cases